<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pqParallelCoordinatesChartDisplayPanel</class>
 <widget class="QWidget" name="pqParallelCoordinatesChartDisplayPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="pqTreeWidget" name="CompositeIndex">
     <column>
      <property name="text">
       <string>Select Block</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="SeriesGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Parallel Coordinates Properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="pqTreeView" name="SeriesList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis">
         <string>This displays the list of line series. You can use the list to edit the series options.</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Line Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="pqColorChooserButton" name="ColorButton">
        <property name="whatsThis">
         <string>Press this button to choose a new color for the line.</string>
        </property>
        <property name="text">
         <string>Choose Color...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ThicknessLabel_3">
        <property name="text">
         <string>Opacity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="Opacity">
        <property name="toolTip">
         <string>&lt;html&gt;Set the opacity of the dataset's geometry.  Artifacts
         may appear in translucent geometry because primatives are not
         sorted.&lt;/html&gt;</string>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="ThicknessLabel">
        <property name="text">
         <string>Line Thickness</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="Thickness">
        <property name="whatsThis">
         <string>Enter the thickness for the line.</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="StyleListLabel">
        <property name="text">
         <string>Line Style</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="StyleList">
        <property name="whatsThis">
         <string>Select the line style for the series.</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash Dot</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dash Dot Dot</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqTreeView</class>
   <extends>QTreeView</extends>
   <header>pqTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>pqTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>pqTreeWidget.h</header>
  </customwidget>
  <customwidget>
   <class>pqColorChooserButton</class>
   <extends>QPushButton</extends>
   <header>pqColorChooserButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
